pipelinewise/cli/pipelinewise.py

```diff
@@ -1233,51 +1228,37 @@ def run_tap(self):
         utils.create_backup_of_the_file(tap_state)
         start_time = datetime.now()
         try:
-            with pidfile.PIDFile(self.tap['files']['pidfile']):
-                target_params = TargetParams(
-                    target_id=target_id,
-                    type=target_type,
-                    bin=self.target_bin,
-                    python_bin=self.target_python_bin,
-                    config=cons_target_config,
-                )
+            target_params = TargetParams(
+                target_id=target_id,
+                type=target_type,
+                bin=self.target_bin,
+                python_bin=self.target_python_bin,
+                config=cons_target_config,
+            )
 
-                transform_params = TransformParams(
-                    bin=self.transform_field_bin,
-                    python_bin=self.transform_field_python_bin,
-                    config=tap_transformation,
-                    tap_id=tap_id,
-                    target_id=target_id,
-                )
+            transform_params = TransformParams(
+                bin=self.transform_field_bin,
+                python_bin=self.transform_field_python_bin,
+                config=tap_transformation,
+                tap_id=tap_id,
+                target_id=target_id,
+            )
 
-                # Run fastsync for FULL_TABLE replication method
-                if len(fastsync_stream_ids) > 0:
-                    self.logger.info(
-                        'Table(s) selected to sync by fastsync: %s', fastsync_stream_ids
-                    )
-                    self.tap_run_log_file = os.path.join(
-                        log_dir, f'{target_id}-{tap_id}-{current_time}.fastsync.log'
-                    )
-                    tap_params = TapParams(
-                        tap_id=tap_id,
-                        type=tap_type,
-                        bin=self.tap_bin,
-                        python_bin=self.tap_python_bin,
-                        config=tap_config,
-                        properties=tap_properties_fastsync,
-                        state=tap_state,
-                    )
+            # Run fastsync for FULL_TABLE replication method
+            if len(fastsync_stream_ids) > 0:
+                self.logger.info(
+                    'Table(s) selected to sync by fastsync/partialsync: %s', fastsync_stream_ids
+                )
+                self.sync_tables()
```

Comment:
If the partial sync breaks, the process continues to singer then finish with success, the process exit code is 0.

Reply:
- [ ] Fix exception handling for multiprocessing